text_extensions_for_pandas/array/arrow_conversion.py

```diff
@@ -24,12 +24,12 @@
 import numpy as np
 import pyarrow as pa
 
-from text_extensions_for_pandas.array import CharSpanArray, TensorArray, TokenSpanArray
+from text_extensions_for_pandas.array import SpanArray, TensorArray, TokenSpanArray
 
 
 class ArrowCharSpanType(pa.PyExtensionType):
```

Comment:
Might be a good idea to rename these as well to keep consistent. I could do that as a followup though.

Reply:
I've made this change too.